views/assets/js/chart/main.js

```diff
@@ -3,10 +3,15 @@ import 'chartjs-adapter-dayjs-4/dist/chartjs-adapter-dayjs-4.esm'
 
 const charts = document.getElementsByClassName('chart-container')
 
-for (const chart of charts) {
-	const chartOptionsTag = chart.querySelector("[data-chart='settings']");
-	const chartCanvasTag = chart.querySelector("[data-chart='canvas']");
+for (const chartElem of charts) {
+	const chartOptionsTag = chartElem.querySelector("[data-chart='settings']");
+	const chartCanvasTag = chartElem.querySelector("[data-chart='canvas']");
 	if (chartOptionsTag != null && chartCanvasTag != null) {
+		chartElem.classList.remove("govuk-visually-hidden")
```

Comment:
I don't think `govuk-visually-hidden` is appropriate here. This is a [special class which sets an element hidden visually but is still available for screen readers](https://www.a11yproject.com/posts/how-to-hide-content/) (see `<h2 class="govuk-visually-hidden">Support links</h2>` in the footer as an example). This makes the Chart/Table tabs available for screen readers and keyboard users, taking up Tab space.